openpype/hosts/fusion/plugins/publish/collect_render.py

```diff
@@ -50,23 +50,45 @@ def get_instances(self, context):
                 continue
 
             family = inst.data["family"]
-            if family != "render":
+            if family not in ["render", "image"]:
                 continue
 
             task_name = context.data["task"]
             tool = inst.data["transientData"]["tool"]
 
+            frame_start = inst.data["frameStart"]
+            frame_end = inst.data["frameEnd"]
+            handle_start = inst.data["handleStart"]
+            handle_end = inst.data["handleEnd"]
+            frame_start_handle = inst.data["frameStartHandle"]
+            frame_end_handle = inst.data["frameEndHandle"]
+
+            label = inst.data["label"]
+            if family == "image":
+                frame = inst.data["creator_attributes"].get("frame")
+                if not frame:
+                    break
+
+                frame = int(frame)
+                label = label.replace(str(frame_start), str(frame))
+                label = label.replace(str(frame_end), str(frame))
+
+                frame_start = frame
+                frame_end = frame
+                frame_start_handle = frame
+                frame_end_handle = frame
```

Comment:
This is hiding a potential issue - up to this point, apparently the image family will have `frameStart`, etc. defined, but not with the right values.

This should be solved instead in `Collect Instances` plug-in so that for image family it defines the frame range DIRECTLY as we intend it so that IF any plug-in would be between that plug-in and this one it also gets the expected single frame - frame range.

Bonus would be that _then_  you'd also NOT need to update the label with string replacement.
String replacements like these are NASTY. If you start frame is 1 and your asset is shot010 but you're now rendering image frame 37 suddenly anything named `shot010` will display as `shot0370` - this is _not_ the type of string replacement you want to be doing.

Anyway, move the collecting of the frame range to the right place (Collect Instances) and you should be good either way.